File: src/test/java/plinkPlonkPlunk/PlinkPlonkPlunkTest.java
package plinkPlonkPlunk;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class PlinkPlonkPlunkTest {

    private PlinkPlonkPlunk plinkPlonkPlunk = new PlinkPlonkPlunk();

    @Test
    public void throwAwayTest() {
        assertThat(plinkPlonkPlunk.findTheSound(1)).isEqualTo("1");
    }
}

File: src/test/kotlin/kotlinCoreTDDTests/plinkPlonkPlunk/PlinkPlonkPlunkTest.kt
package kotlinCoreTDDTests.plinkPlonkPlunk

import kotlinCoreTDD.plinkPlonkPlunk.PlinkPlonkPlunk
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class PlinkPlonkPlunkTest {

    @Test
    fun returns_Plink_for_3() {
        assertThat(PlinkPlonkPlunk().findTheSound(3)).isEqualTo("Plink")
    }

}